introduction.md

```diff
@@ -6,3 +6,7 @@ This repository contains a list of my rules for the Deadly Vipers dojo
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?